README.md

```diff
@@ -1,13 +1,116 @@
 # 💪 strong-config
 
+<!-- markdownlint-disable line-length -->
+[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master) [![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+<!-- markdownlint-enable line-length -->
+
 Secret management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
```

Comment:
I know this was here before, but I do think we should put **config** first and **secrets** second. Because it has utility even if you weren't using it for managing secrets, plus it's called strong-config :)

```suggestion
Simple and secure config management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
```

Reply:
Happy to put `simple`. Not sure about the `secure` - it's misleading imo as it reads as the management is secure.

`Simple config-with-secrets management for NodeJS` ?

Reply:
my argument would be that it _is_ more secure in a few ways:
- It's harder to accidentally check in secrets
- It's using SOPS which makes using strong cryptography much easier

Reply:
Hm yeah. Your first point is more safety, not security.

Still feeling weird about `secure management` as a first-sentence selling point. I'll check the SOPS repo and see how they market themselves to get some inspiration